http-clients/src/main/java/com/palantir/remoting3/clients/ClientConfigurations.java

```diff
@@ -50,6 +51,14 @@ private ClientConfigurations() {}
      * empty/absent configuration with the defaults specified as constants in this class.
      */
     public static ClientConfiguration of(ServiceConfiguration config) {
+        return of(config, DEFAULT_RPC_MODE);
+    }
+
+    /**
+     * Creates a new {@link ClientConfiguration} instance from the given {@link ServiceConfiguration} and
```

Comment:
better: `Like {@link of(ServiceConfiguration)}, but using the explicitly given NodeSelectionStrategy.`